assignments/prototypes.js

```diff
@@ -15,6 +15,15 @@
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
 
+function GameObject(objAtt) {
+  this.createdAt = objAtt.createdAt;
+  this.dimension = objAtt.dimension;
```

Comment:
You just forgot the "s". Otherwise, your prototypes are perfect.

```suggestion
  this.dimensions = objAtt.dimensions;
```